x-pack/legacy/plugins/transform/public/app/sections/create_transform/components/step_details/step_details_summary.tsx

```diff
@@ -34,40 +34,42 @@ export const StepDetailsSummary: SFC<StepDetailsExposedState> = React.memo(
 
     return (
       <Fragment>
```

Comment:
Since there's now a wrapping `div` element, you can remove the `Fragment` tag.

Reply:
:+1: Removed in 9e4e6c8